include/deal.II/base/template_constraints.h

```diff
@@ -599,8 +599,6 @@ template <typename T> struct EnableIfScalar<std::complex<T> >
 
 
 
-
-
```

Comment:
:-D *oops* Happens if you refactor twice *cough*. Do you want to have your empty lines back? (But be warned - they will never be the same again...)